packages/ipfs-http-client/src/dht/find-peer.js

```diff
@@ -9,23 +9,23 @@ module.exports = configure(api => {
   return async function findPeer (peerId, options = {}) {
     options.arg = `${Buffer.isBuffer(peerId) ? new CID(peerId) : peerId}`
 
-    const res = await api.post('dht/findpeer', {
+    const res = await api.ndjson('dht/findpeer', {
```

Comment:
Since this was changed from a `.post` it needs to include `method: 'POST'` in the options. 
See https://github.com/ipfs/js-ipfs/blob/master/packages/ipfs-http-client/src/dht/find-provs.js#L10-L15

Reply:
Good point - I think @lidel was going to PR the client to change all methods to post.

Reply:
I can do that now though..

Reply:
Taking shape here: https://github.com/ipfs/js-ipfs/pull/2977